src/dataBuilderFactory.ts

```diff
@@ -66,6 +66,17 @@ function isEmpty(v: any): boolean {
     return false;
 }
 
+function trim(value: FieldValue): FieldValue {
+    if (value) {
+        if (isString(value)) {
+            return value.trim();
+        } else if (Immutable.Iterable.isIterable(value)) {
+            return value.map((v) => trim(v));
```

Comment:
Shouldn't the values inside iteration also be protected with isString()? In case they are null or numbers or something?

Reply:
but it will be in the recursive call